docs/spelling_wordlist.txt

```diff
@@ -347,3 +347,14 @@ initramfs
 efi
 partprobe
 lfs
+initramdisk
```

Comment:
Not related to this repository or commit. But this means some of these you would not need in software-factory's spelling word list. So make sure you remove them from software factory. Otherwise I am ok with it